src/partners/kado.ts

```diff
@@ -158,3 +169,19 @@ export function processKadoTx(rawTx: unknown): StandardTx {
     }
   }
 }
+
+function getFiatPaymentType(tx: KadoTx): FiatPaymentType | null {
+  if (!('paymentMethod' in tx)) return null
```

Comment:
Can we throw if we don't match a paymentMethod. That way if we add new payment methods to Kado it enforces that we add them here.